.travis.yml

```diff
@@ -159,20 +149,16 @@ jobs:
     - stage: test
       env:
         - PROJECT=Firestore PLATFORM=iOS METHOD=pod-lib-lint
-      before_install:
-        - ./scripts/if_changed.sh ./scripts/install_prereqs.sh
       script:
         # Eliminate the one warning from BoringSSL when CocoaPods 1.6.0 is available.
         # The travis_wait is necessary because the command takes more than 10 minutes.
         - travis_wait 30 ./scripts/if_changed.sh ./scripts/pod_lib_lint.rb FirebaseFirestore.podspec --platforms=ios --allow-warnings --no-subspecs
 
     # pod lib lint to check build and warnings for static library build - only on cron jobs
     - stage: test
-      if: type = cron
+#      if: type = cron
       env:
-        - PROJECT=Firebase PLATFORM=iOS METHOD=pod-lib-lint
-      before_install:
-        - ./scripts/install_prereqs.sh
+        - PROJECT=FirebasePllCron1 PLATFORM=iOS METHOD=pod-lib-lint
```

Comment:
Changing the `PROJECT` itself should affect `./scripts/if_changed.sh`. Looks like it will fallback to `*)` option and fail.